redox/response_flowsheet.go

```diff
@@ -67,29 +67,41 @@ func NewFlowsheet() models.NewFlowsheet {
 	return flowsheet
 }
 
+type FlowsheetSettings struct {
+	PreferredBGUnits string
+	ICode            bool
+}
+
+type Observation struct {
+	Value       string
+	ValueType   string
+	Units       *string
+	Description string
+}
+
 // PopulateSummaryStatistics populates a flowsheet with patient summary statistics. If summary statistics are not available,
 // the flowsheet items will be populated with 'NOT AVAILABLE'.
-func PopulateSummaryStatistics(patient clinics.Patient, clinic clinics.Clinic, flowsheet *models.NewFlowsheet) {
-	var cgmStats *clinics.PatientCGMStats
-	var bgmStats *clinics.PatientBGMStats
+func PopulateSummaryStatistics(patient clinics.Patient, settings FlowsheetSettings, flowsheet *models.NewFlowsheet) {
 	if patient.Summary != nil {
-		cgmStats = patient.Summary.CgmStats
-		bgmStats = patient.Summary.BgmStats
+		if patient.Summary.CgmStats != nil && !patient.Summary.CgmStats.Dates.LastUpdatedDate.IsZero() {
```

Comment:
I think this logic should be put behind a feature flag, because it changes how it works for existing clinics.

Reply:
confirmed that this should be default behavior and not behind a feature flag.